src/OrleansTestingHost/TestCluster.cs

```diff
@@ -278,6 +287,8 @@ public SiloHandle StartAdditionalSilo()
                 baseSiloPort, 
                 baseGatewayPort);
 
+            configurationModification?.Invoke(nodeConfig);
+
```

Comment:
These are remnants of old code, but you are not really using the overload of StartAdditionalSilo that takes an action, right?